app/main.py

```diff
@@ -1,34 +1,62 @@
+from typing import List, Tuple
+
+
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(self, row: int,
+                 column: int,
+                 is_alive: bool = True) -> None:
+        self.row = row
+        self.column = column
+        self.is_alive = is_alive
 
 
 class Ship:
-    def __init__(self, start, end, is_drowned=False):
-        # Create decks and save them to a list `self.decks`
-        pass
+    def __init__(self, start: Tuple[int, int],
+                 end: Tuple[int, int],
+                 is_drowned: bool = False) -> None:
+        self.start = start
+        self.end = end
+        self.is_drowned = is_drowned
+        self.decks = self.create_decks()
 
-    def get_deck(self, row, column):
-        # Find the corresponding deck in the list
-        pass
+    def create_decks(self) -> List[Deck]:
+        if self.start[0] == self.end[0]:
+            return ([Deck(self.start[0],
+                          column) for column in
+                     range(self.start[1], self.end[1] + 1)])
+        else:
+            return ([Deck(row, self.start[1]) for row in
+                     range(self.start[0], self.end[0] + 1)])
```

Comment:
Use loop inside loop instead of if-else:
```
return [
    Deck(row, column)
    for row in range(self.start[0], self.end[0] + 1)]:
    for column in range(self.start[1], self.end[1] + 1)]
]
```